diff --git a/node_modules/@oraichain/cw-simulate/dist/fork.js b/node_modules/@oraichain/cw-simulate/dist/fork.js
index 3b6412a..afdec00 100644
--- a/node_modules/@oraichain/cw-simulate/dist/fork.js
+++ b/node_modules/@oraichain/cw-simulate/dist/fork.js
@@ -121,7 +121,6 @@ const downloadState = async (rpc, contractAddress, writeCallback, startAfter, li
         try {
             const { models, pagination } = await cosmwasmClient.getAllContractState(contractAddress, nextKey, limit);
             writeCallback(models);
-            console.log('next key: ', Buffer.from(pagination.nextKey).toString('base64'));
             if (!pagination.nextKey || pagination.nextKey.length === 0) {
                 return;
             }
